team-builder/src/components/Team.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Team = props => {
```

Comment:
I would recommend thinking a bit more on the naming of your components. I think `TeamList` would be a better name for this component.